coalib/collecting/Collectors.py

```diff
@@ -82,7 +82,9 @@ def collect_files(file_paths, log_printer, ignored_file_paths=None,
         collected_files, file_globs_with_files = [], []
 
     _warn_if_unused_glob(log_printer, file_paths, file_globs_with_files,
-                         "No files matching '{}' were found.")
+                         "No files matching '{}' were found. "
+                         'If this rule is not required, you can remove it '
```

Comment:
Can you maintain uniformity and use either double or single quotes? Double looks good here.